.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E501
+ignore = E501,W503
```

Comment:
It doesn't look like we're following `W503` closely so I ignored it (it was already failing in the last merged PR #71)

Reply:
For more info: https://www.flake8rules.com/rules/W503.html